Library/Formula/srmio.rb

```diff
@@ -15,7 +15,7 @@ class Srmio < Formula
 
   def install
     if build.head?
-      system "chmod u+x genautomake.sh"
+      chmod 1, "genautomake.sh"
```

Comment:
I _think_ this should be:
```ruby
chmod "u+x", "genautomake.sh"`
```
But I'm double checking with the other maintainers. Haven't seen much of Ruby's `chmod` method being invoked in a non-octal mode.